packages/jingoo/jingoo.1.2.21/descr

```diff
@@ -0,0 +1 @@
+Template engine almost compatible with Jinja2(python template engine)
```

Comment:
Could you use the `synopsis` field instead of a descr file?

Reply:
Thank you for your advice!